### PR TITLE
Update Metadata

### PR DESCRIPTION
**Pull request by willfarrell**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -43,35 +43,39 @@ For browser requests all you need to do is let us know your domain name and we c
...
+  - Select By: `Id`, `DOI`, `DatasetName`, `MonitoringLocationID`, `MonitoringLocationName`, `MonitoringLocationLatitude`, `MonitoringLocationLongitude`, `MonitoringLocationHorizontalCoordinateReferenceSystem`, `MonitoringLocationHorizontalAccuracyMeasure`, `MonitoringLocationHorizontalAccuracyUnit`, `MonitoringLocationVerticalMeasure`, `MonitoringLocationVerticalUnit`, `MonitoringLocationType`, `ActivityType`, `ActivityMediaName`, `ActivityStartDate`, `ActivityStartTime`, `ActivityEndDate`, `ActivityEndTime`, `ActivityDepthHeightMeasure`, `ActivityDepthHeightUnit`, `SampleCollectionEquipmentName`, `CharacteristicName`, `MethodSpeciation`, `ResultSampleFraction`, `ResultValue`, `ResultUnit`, `ResultValueType`, `ResultDetectionCondition`, `ResultDetectionQuantitationLimitMeasure`, `ResultDetectionQuantitationLimitUnit`, `ResultDetectionQuantitationLimitType`, `ResultStatusID`, `ResultComment`, `ResultAnalyticalMethodID`, `ResultAnalyticalMethodContext`, `ResultAnalyticalMethodName`, `AnalysisStartDate`, `AnalysisStartTime`, `AnalysisStartTimeZone`, `LaboratoryName`, `LaboratorySampleID`
+  - Filter By: `DOI`, `LocationId`, `RegionId`
```
Comment (mandy-kudos) [inline docs/README.md]: Should this be MonitoringLocationID so it matches?

Comment (willfarrell) [reply docs/README.md]: `LocationId` != `MonitoringLocationID`

`LocationId` == to the `Id` from `/Locations`, we can remove for now. Add in later if someone asks

File: docs/README.md [hunk 2]
```diff
@@ -97,6 +98,7 @@ OData accepts certain query parameters. The ones supported by this API are:
       - Watersheds/Drainage Areas: `watershed.oda.*`,`watershed.mda.*`,`watershed.sda.*`,`watershed.ssda.*` (Future)
       - Water: `waterbody.marine.*`, `waterbody.greatlakes.*`, `waterbody.lakes.*`, `waterbody.rivers.*` (Future)
     - Bounding box `$filter=LongitudeNormalized gt '-102.01' and LongitudeNormalized lt '-88.99' and LatitudeNormalized gt '49' and LatitudeNormalized lt '60'`
+  - Functions: `$filter=contains(DOI, '10.25976')`, `$filter=startwith(DOI, 'https://')`, `$filter=endswith(DOI, 'xxxx-xxxx')`
```
Comment (willfarrell) [inline docs/README.md]: FYI all DOI will start with `https://doi.org/10.25976/`

File: docs/README.md [hunk 3]
```diff
@@ -129,15 +124,15 @@ Get the citation and licence for a dataset:
 ```bash
 curl -G -H 'x-api-key: PRIVATE-API-KEY' \
      https://api.datastream.org/v1/odata/v4/Metadata \
-     --data-urlencode "\$select=Name,Licence,Citation,Doi,Version" \
-     --data-urlencode "\$filter=DatasetId eq '0000-00000-00000-00000'" \
+     --data-urlencode "\$select=DOI,DatasetName,Licence,Citation,Version" \
+     --data-urlencode "\$filter=DOI eq '10.25976/xxxx-xxxx'" \
```
Comment (willfarrell) [inline docs/README.md]: Will need to update to use `$filter=endswith(DOI, 'xxxx-xxxx')`

**Pull request merged**